frontend/src/components/jobs/result/utils/sections.jsx

```diff
@@ -111,6 +112,10 @@ export function JobActionsBar({ job }) {
 }
 
 export function JobInfoCard({ job }) {
+  const process_time2 = new Date(job.process_time * 1000)
```

Comment:
is not a good way to add numbers to variables. Please find a more explicative way to call it